ansible/roles/test/tasks/acltb.yml

```diff
@@ -115,17 +115,8 @@
             - testbed_type=\"{{ testbed_type }}\"
             
   always:
-    # Copy ACL config to the switch
-    - name: Copy ACL config file to the DUT
-      copy: src="roles/test/tasks/acl/{{ item }}" dest="/tmp/"
-      with_items:
-        - "acltb_test_rules-del.json"
-      
     - name: Clean up ACL rules.
-      vars:
-        command_to_run: "acl-loader update full /tmp/acltb_test_rules-del.json"
-        errors_expected: false
-      include: roles/test/tasks/run_command_with_log_analyzer.yml
+      command: acl-loader delete

```

Comment:
Why this change as compared to the previous one?. This will delete the entire set of rules but the previous approach deletes the rules based on the json file.

Reply:
@prsunny  acltb_test_rules-del.json file is a empty file. So using either json or 'delete' command performs the same action. Using acl-loader delete is straight forward and reduces the number of lines of code, so made this change. 

Additionally I verified acl-loader delete command doesn't remove the DATAACL applied on the interfaces.